src/main/kotlin/no/nav/syfo/behandlendeenhet/EnhetService.kt

```diff
@@ -72,12 +74,15 @@ class EnhetService(
 
     fun updatePerson(personIdent: PersonIdentNumber, isNavUtland: Boolean): Person? {
         val pPerson = database.updatePerson(personIdent, isNavUtland)
```

Comment:
Kanskje man skulle gjort .toPerson() her (i stedet for i return ), så kan man sende inn Person-domeneobjekt til producer som @JMLindseth foreslår under

Reply:
Ja, men da mister man `updatedAt`-feltet som vi ønsker å sende på kafka. Alternativet er å bare sende inn to parametere `personident` og `updatedAt` hver for seg🤷🏼‍♂️

Reply:
Evt sende en `KBehandlendeEnhetUpdate` inn i selve funksjonen (som blir wrapperen til de feltene uansett)